render_html_test.go

```diff
@@ -127,8 +159,9 @@ func TestRenderBlock(t *testing.T) {
 		Layout:    "layout",
 	})
 
+	var err error
```

Comment:
This var interferes with the next statement block... one of them should be renamed.